chart/infra-server/templates/certificate.yaml

```diff
@@ -1,4 +1,4 @@
-{{ if ne .Values.deployment "local" -}}
+{{- if .Capabilities.APIVersions.Has "networking.gke.io/v1/ManagedCertificate" -}}
```

Comment:
This is going to create the managedcertificate on infra PR instances with dev values. How will this affect the dev instance? If we are not sure, then best to avoid.

Reply:
The managed certificate will be stuck in Provisioning state for infra PR instances, because the certificate challenge cannot be reached on the `{{ .Values.hosts.primary }}` or `{{ .Values.hosts.secondary }}`.

No impact on the dev cluster other than some failed requests from GCP.

Reply:
OK https://github.com/stackrox/infra/pull/1015/commits/ce8e555ae19ce0136a884d69bf213289654fffc0 :